_includes/code/animation/flying_transition/pubspec.yaml

```diff
@@ -1,4 +1,4 @@
-name: shared_element_transition
+name: flying_shared_element_transition
```

Comment:
Suggestions? At one point I called this example shared_element_transition, but both this and the radial examples are shared_element_transitions.

Reply:
Maybe you could call this demo "heroic_flight" since (if I'm right about which demo you're referring to) it's just the basic hero flight. It may be asking too much to try and come up with names that can be read like descriptive expressions.

If that made sense then perhaps the other demo would be expanding_heroic_flight.

Reply:
I don't love how vague "hero" really is. To many people, a "hero" is a banner image on a web page.